.gitignore

```diff
@@ -8,3 +8,5 @@ tmp
 /*.sh
 /TODO
 pkg/manifests/bindata.go
+
+!hack/dockerfile_install_support.sh
```

Comment:
Is this still needed since we moved the script to `hack/` directory?

Reply:
Not needed any longer. Fixed in next push.